feature-libs/product-configurator/rulebased/components/service/configurator-storefront-utils.service.ts

```diff
@@ -254,7 +254,21 @@ export class ConfiguratorStorefrontUtilsService {
    * @returns {string} - generated group ID
    */
   createOvGroupId(prefix: string, groupId: string): string {
-    return `id${prefix}${groupId}-ovGroup`;
+    return prefix ? prefix + '--' + groupId + '-ovGroup' : groupId + '-ovGroup';
```

Comment:
We need to document the changed behaviour

Reply:
Done!